scripts/generate_duration.py

```
@@ -14,50 +20,63 @@
-            continue
-
-        # Add to the test duration dictionary
-        test_durations[test_name] = last_column
-
-# Write the test durations to the new test duration file
+# Write the test durations to the output file
 with open(NEW_TEST_DURATION, 'w') as file:
     json.dump(test_durations, file, indent=3)
 
-print(f"Test durations have been written to {NEW_TEST_DURATION}")
+print(f"\nSummary:")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)